spec/features/guide_history_spec.rb

```diff
@@ -59,3 +63,23 @@ def events
       .reverse
   end
 end
+
+
+RSpec.describe "Guide history", type: :feature do
+  scenario "viewing previous editions" do
+    guide = create(:published_guide)
+    guide.editions << build(:edition, version: 2)
+
+    visit edition_comments_path(guide.reload.latest_edition)
+
+    expect_edition_to_be_open("Version #2")
+
+    click_link "Version #1"
+
+    expect_edition_to_be_open("Version #1")
+  end
+
+  def expect_edition_to_be_open(title)
+    expect(page).to have_css(".open-edition .panel-heading", text: title)
```

Comment:
Yeah, the CSS class `open-edition` makes it easier to test. Personally I'm comfortable with it. It's a pretty semantic block name and isn't likely to change much and so will be less likely to break the test than, for example, we check which type of chevron is used to determine open-ness. We make little compromises all over to increase the testability of code.

I'm open to suggestions - what would you prefer?